contracts/Strategy.sol

```diff
@@ -254,17 +257,18 @@ contract Strategy is BaseStrategy {
                 (uint256 _mToken, ) = MMFarmingPool(mmFarmingPool).userInfo(mmFarmingPoolId, address(this));
                 require(_mToken >= _mvGap, '!insufficientMTokenInMasterChef');
```

Comment:
can this be made to always revert by someone? perhaps by airdroppig in some tokens to confuse calculations?

Reply:
Also i think this ruins your liquidateall solution. it will revert if we ask for more than our assets

Reply:
There is no way to airdrop to Mushrooms farming pool to affect the `userInfo` output for this strategy. If the attacker want to airdrop `want` token or `mmVault` share token directly to strategy, I do not see the economic incentive here because what is airdropped will be taken as strategy asset to get returned to yVault